pkg/postgres/schema/process_indicators.go

```diff
@@ -58,13 +58,13 @@ const (
 type ProcessIndicators struct {
```

Comment:
Just to clarify something to make sure we do not need a migration.  If memory serves these updates were NOT part of 3.74.  Is that correct?  If that is the case we do not need a migration to remove those indexes.

Reply:
Nothing related to processes listening on ports went into central for 3.74, except the feature flag. These updates were not a part of 3.74.